cmd/commands/client/command_client.go

```diff
@@ -97,16 +101,24 @@ type Command struct {
 	connectionManager connection.Manager
 	httpAPIServer     tequilapi.APIServer
 	checkOpenvpn      func() error
+	originalLocationCache     location.Cache
 }
 
-// Start starts Tequilapi service - does not block
+// Start starts Tequilapi service, fetches location
 func (cmd *Command) Start() error {
 	log.Info("[Client version]", version.AsString())
 	err := cmd.checkOpenvpn()
 	if err != nil {
 		return err
 	}
 
+	originalLocation, err := cmd.originalLocationCache.RefreshAndGet()
+	if err != nil {
+		log.Warn("Failed to detect country", err)
+	} else {
+		log.Info("Country detected: ", originalLocation.Country)
```

Comment:
Logs of original and vpn country could differ - i.e. "Original country detected: Lithuania", "VPN country detected: China" - it would be easier to debug.

Reply:
added "original" to log message